ansible_base/authentication/utils/authentication.py

```diff
@@ -89,13 +96,11 @@ def determine_username_from_uid(uid: str = None, authenticator: Authenticator =
 
 
 def get_or_create_authenticator_user(
-    username: str, authenticator: Authenticator, user_details: dict = dict, extra_data: dict = dict
+    uid: str, authenticator: Authenticator, user_details: dict = dict, extra_data: dict = dict
```

Comment:
Can we add a doc blurb here about what `uid` is?

Reply:
https://github.com/python-social-auth/social-app-django/blob/af9ea55bbc057018056b1789c23c663e7c89d483/social_django/models.py#L45